content/learning-paths/cross-platform/kleidiai-explainer/_next-steps.md

```diff
@@ -0,0 +1,24 @@
+---
+next_step_guidance: Check out the KleidiAI further reading to further understand what it is capable of. 
+
+recommended_path: /learning-paths/servers-and-cloud-computing/llama-cpu/
+
+further_reading:
```

Comment:
Worth pointing to KleidiCV here?

Reply:
I personally don't think so as that is for a different set of users doing graphics. KleidiAI is focusing on LLM-based devs

Reply:
No worries, let's save the excitement for another LP